game/modes.py

```diff
@@ -0,0 +1,143 @@
+from .board import board_matrix as initial_board, get_options, show, update_board
+from .status import check_status
+from .player import get_current_player
+import logging
+import random
+import copy
+
+
+logger = logging.getLogger(__name__)
+
+
+def _find_winning_cell(board_matrix, available_options, sign):
+    '''
+    Creates a copy of the current state of the board and tries all the available options with the sign argument
+    and checks if any choice wins, returns True or False if there is a winning choice, along with the the choice itself.
+    '''
```

Comment:
PEP: Triple double-quoted strings should be used for docstrings.

I would rephrase the comment as it is a bit too long and becomes hard to follow :)

```suggestion
    '''
    Creates a copy of the current state of the board and tries all the available options with the sign argument.
    Checks if any choice wins.
    Returns True or False if there is a winning choice, along with the the choice itself.
    '''
```